truefoundry.yaml

```diff
@@ -401,3 +395,51 @@ components:
         persistence:
           size: 5Gi
       architecture: standalone
+
+  # Whisper
+  - name: cas-whisper
+    env:
+      WHISPER_PORT: 8000
+      WHISPER__MODEL: Systran/faster-distil-whisper-large-v3
+      WHISPER__INFERENCE_DEVICE: auto
```

Comment:
These double underscores in key names are correct?

Reply:
yes, they are from the dockerfile, checked.

Reply:
https://github.com/fedirz/faster-whisper-server/blob/master/Dockerfile.cpu